Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:buster-slim
+
+RUN apt-get update
+RUN apt-get install -y \
+    build-essential \
+    curl \
+    gnupg \
+    hugo=0.47.1-2
+
+# Install node.js 8.x (LTS at time of writing) from official package.
+RUN curl --silent --location https://deb.nodesource.com/setup_8.x | bash -
+RUN apt-get -y update
+RUN apt-get install -y nodejs
+
+ADD . /src
```

Comment:
I think the current practice to to use `COPY` instead of `ADD` because it's more explicit.